src/coffee/import.coffee

```diff
@@ -43,10 +43,12 @@ class Import
     options.importFormat = options.importFormat || "csv"
     options.csvDelimiter = options.csvDelimiter || ","
     options.encoding = options.encoding || "utf-8"
+    options.mergeCategoryOrderHints = !!options.mergeCategoryOrderHints
```

Comment:
Please use `Boolean` instead, it's a bit more expressive. Same in the `run.coffee` file.